src/adapter/src/catalog.rs

```diff
@@ -6283,6 +6368,30 @@ impl Catalog {
                     );
                     builtin_table_updates.extend(state.pack_item_update(id, 1));
                 }
+                Op::Comment {
```

Comment:
Something needs to handle what happens to comments on drop/alter of the objects or columns. Are there dependencies we need to track? Can we just drop all comments on drop and not track dependencies?